docs/source/conf.py

```diff
@@ -76,8 +71,12 @@
     "external_links": [],
     "github_url": "https://github.com/NCAR/musica",
     "navbar_end": ["version-switcher", "navbar-icon-links"],
-    "pygments_light_style": "tango",
-    "pygments_dark_style": "monokai",
+    "switcher": {
+        "json_url": "https://ncar.github.io/musica/_static/switcher.json",
```

Comment:
```suggestion
        "json_url": "https://ncar.github.io/musica/branch/version/_static/switcher.json",
```

Temporary test of switcher

Reply:
this works.